.github/workflows/ci.yaml

```diff
@@ -185,9 +185,70 @@ jobs:
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
       - name: Build and push
         id: docker_build
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
+        with:
+          file: dse-68/Dockerfile.${{ matrix.jdk-version }}
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          platforms: linux/amd64
+          target: dse68
+
+  publish-dse-ubi:
+    name: Publish DSE UBI image
+    if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
+    needs: build-dse
```

Comment:
Shouldn't this be `needs: build-dse-68-ubi7`

Reply:
Yep!

Reply:
Actually, no this is correct, or at least the intent. `build-dse` is a job that builds 1 image (Ubuntu based JDK8 image) and runs the integration tests. The assumption here (and it is potentially a poor assumption) is that as long as 1 of the DSE images passes the DSE tests, they all should. So there is only 1 test job (`build-dse`) and if that passes, the image building jobs for all DSE variants are green lit to do the image building as those jobs don't run tests.